src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/NdmisPerformanceReportJobConfiguration.kt

```diff
@@ -44,11 +44,16 @@ class NdmisPerformanceReportJobConfiguration(
   @Value("\${spring.batch.jobs.ndmis.performance-report.chunk-size}") private val chunkSize: Int,
 ) {
   companion object : KLogging()
+  private val pathPrefix = "dfinterventions/dfi/csv/reports/"
```

Comment:
_not a blocker_
This is tightly coupled with the environment: https://github.com/ministryofjustice/cloud-platform-environments/blob/26a924086f42f2edff4502734d0dcdf3a0961364/namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-prod/resources/reporting-landing-access.tf#L58

To be independent from this path changing in the infrastructure, as a next iteration, we could read it from a secret

Reply:
We could also change 

https://github.com/ministryofjustice/data-engineering-data-extractor/blob/v1.1.1/modules/transfer_local_to_s3.sh#L7

https://github.com/ministryofjustice/hmpps-interventions-service/blob/main/helm_deploy/hmpps-interventions-service/templates/cronjob-data-extractor-analytics.yaml#L38-L42

to align with the dual "bucket_name", "prefix" configuration that's more aligned with the AWS SDK